instance-id-handler/instanceIDhandler.go

```diff
@@ -0,0 +1,161 @@
+package instanceIDhandler
```

Comment:
```suggestion
package instanceidhandler
```
Also, please change the directory name to `instanceidhandler` since this is the package naming convention in go